tex2pdf-tools/tex2pdf_tools/directives/directives.py

```diff
@@ -25,15 +25,15 @@ def serialize_data(data: dict, format: str) -> str:
     Returns:
         str: Serialized string.
     """
-    if format == "yaml":
-        return yaml.dump(data)
-    elif format == "json":
-        return json.dumps(data, indent=4)
-    elif format == "toml":
-        return toml.dumps(data)
-    else:
-        raise ValueError("Unsupported format. Use 'yaml', 'json', or 'toml'.")
-
+    match format:
+        case "yaml":
+            return yaml.dump(data)
+        case "json":
+            return json.dumps(data, indent=4)
+        case "toml":
+            return toml.dumps(data)
+        case _:
```

Comment:
In the zerozeroreadme code we have a bit more options
```
        match ext:
            case ".yml" | ".yaml":
                loader = YAML()
                zzrm = loader.load(data)
            case ".json" | ".jsn" | ".ndjson":
                zzrm = json.loads(data)
            case ".toml":
                zzrm = toml.loads(data)
```
Should we standardize that?